### DISCUSSION TITLE
Setting up Cron Job

### DISCUSSION THREAD

**Discussion by zishanali06** [coccoinomane/crabada.py#101] [Q&A] [answered]
Hey,

So I added the code to the crontab but it doesnt seem to do anything at all. Any way I can output the logs from what the cron is doing?

Also I dont think I have my /path/to/python done properly, this is what I did for the cron below:

*/1 * * * * cd /Users/myuser/Downloads/crabada.py-master && python3 -m bin.looting.closeLoots (my wallet)

i also tried:

*/1 * * * * cd /Users/myuser/Downloads/crabada.py-master && '/Applications/Python 3.10' -m bin.looting.closeLoots (my wallet)

I am thinking that python3 path(should I make it to the python folder or to like a exact file for this path?) is wrong thats why its not working. Any input would help :).

Everything else working like a charm, now just want to get it on the cron so it can go automatically, I am using a Mac btw

**Comment by loayei** [comment, marked as answer]
Thats not the correct python path. You need to get it using terminal.

**Reply by zishanali06** [reply]
Thanks for that! I used which python3 to get the path now the line of code that goes into crontab looks like this below:

2,12,22,32,42,52 * * * * cd /Users/myuser/Downloads/crabada.py-master && /Library/Frameworks/Python.framework/Versions/3.10/bin/python3 -m bin.mining.reinforceDefense (my address)

Then I exited crontab with control + X and then pressed enter to save. Then I rechecked the file to make sure it is saved there and it was. But the line of code is not firing as the mines are not being reinforced pal. Anything I have to do to activate it?? Also any way I can make this print some log files so maybe I can see what the issue is? Or do I just have the line of code wrong?

Edited: So this line of code, if I use like this in terminal it works so I know I got the path right now:  
/Library/Frameworks/Python.framework/Versions/3.10/bin/python3 -m bin.mining.reinforceDefense (my address)
But its still not working in the crontab.

**Reply by loayei** [reply]
Crontab do have logs. I believe u just have to type (mail) in the terminal.

**Reply by zishanali06** [reply]
Ok but logs aside, any clue why the crontab cronjob not working? I have the line of code proper now right?

Also will the logs show me like the output of the bot? Or just the commands if I do with mail?

**Reply by zishanali06** [reply]
I think I might have figured it out! I was copying the line back from here and I forgot I changed my real user to myuser to post on here and forgot to change it back to my real username. I think now it will work but need to wait 2 hours to test it. Ill update once I try again
